File: moderate/sameend.scala
object Main extends App {
  val source = scala.io.Source.fromFile(args(0))
  val lines = source.getLines.filter(_.length > 0)

  for (l <- lines) {
    val s = l.split(",")
    println(if (s(0).endsWith(s(1))) 1 else 0)
  }
}

File: moderate/sameend.clj
(require '[clojure.string :as str])
(doseq [item (for [st (remove empty? (str/split-lines (slurp (first *command-line-args*))))]
  (#(.endsWith (first %) (second %))
(str/split st #",")))] (prn (if item 1 0)))
